### ISSUE TITLE
Activation Functions from Scratch in Keras

### ISSUE THREAD

**Issue by Agrover112** [Python-World/Python_and_the_Web#101]
#31 Activation Functions from Scratch in Keras

In the following project, we experiment and  implement activation functions commonly used in deep learning libraries from scratch using Keras and Tensorflow

## Type of issue
- NOTE: *These boxes can be checked using **`[X]`***

- [x] Feature (New Script)
- [ ] Bug
- [ ] Documentation

## Checklist:

- [x] I have read the project guidelines.
- [x] I have checked all the existing projects, before submitting a new project issue.
- [x] I have checked previous issues to avoid duplicates.
- [x] This issue will be meaningful for the project.

**Comment by kchatr** [comment]
Hey! Could I implement the sigmoid activation function?

**Comment by AdityaJ7** [comment]
Cool @Agrover112 assigning it to you

**Comment by AdityaJ7** [comment]
> Hey! Could I implement the sigmoid activation function?

Hey @kchatr you can implement it if @Agrover112 ain't but I guess he will be covering Sigmoid cause it is commonly used

**Comment by Agrover112** [comment]
@kchatr  Hey this project focuses on implementing some lesser known activation functions. Mainly those based on RELU and it's variants

**Comment by AdityaJ7** [comment]
> @kchatr Hey this project focuses on implementing some lesser known activation functions. Mainly those based on RELU and it's variants

So you are not implementing Sigmoid?

**Comment by Agrover112** [comment]
Nope the aim was to implement RELU based variants . :(

**Comment by AdityaJ7** [comment]
> Nope the aim was to implement RELU based variants . :(

Ok

**Comment by AdityaJ7** [comment]
@kchatr  create a separate issue concerned with the implementation of Sigmoid function

**Comment by Agrover112** [comment]
I understand why you may want sigmoid by the project takes a different approach mainly relu based variants , which are bit different to sigmoid and not good performance wise @kchatr
